docs/features/web-api-template.md

```diff
@@ -23,16 +23,31 @@ When installed, the template can be created with shortname: `arcus-webapi`:
 ## Features
```

Comment:
Actually I would go from:
- Feature 
	- Version 0.1
- Config
	- Version 0.1
- Feature 
	- Version 0.2
- Config
	- Version 0.2

To
- Feature 
	- Version 0.1
	- Version 0.2
- Config
	- Version 0.1
	- Version 0.2

Reply:
But then you have to look at two places to find the available functionality. Now the configuration and security and possibly other headers are sub headers of each version.

Reply:
I don't really agree so let's ask a 3rd person.

Currently have this @SamVanhoutte, what do you think?
![image](https://user-images.githubusercontent.com/4345663/77670437-7ce67400-6f86-11ea-8b0a-402f53d831b6.png)

Or should we use following approach?

Feature
- Version 0.1
- Version 0.2
Config
- Version 0.1
- Version 0.2

Reply:
Any feedback @SamVanhoutte ?

Reply:
Not really following 100%, but I'd think that you typically are only interested in the version you are using, so why is there no proposal that has 
Version 0.2
- Feature
- Config
Version 0.1
- Feature 
- Config

Might have missed the point, so bear with me.  Also I believe the most recent version should be at the top?

Reply:
That's certainly an option but that assumes you'll duplicate the information from v0.1 in v0.2 then or not?

So what I actually suggest is to move this to the URL :)

`/features/templates/web-api` goes to latest version which is noted at top via badge. At the bottom you can find a list of old docs.
`/v0.1/features/templates/web-api` provides the old docs 

Then the docs are scoped per page and have capability to go back.

Thoughts?

Reply:
That sounds logical, indeed.  
`As a user`, I just want to see the documentation of the version I'm using and most of the time I don't care about older versions.  Again: up to you two

Reply:
Let's use URLs then!

Reply:
This is actually a problem for every Arcus project.

Reply:
An opportunity you mean ;)

That's correct but nothing we can't fix. Let's start doing that here and if it's good we can roll it out to others